app/containers/App.js

```diff
@@ -51,23 +56,13 @@ class App extends React.Component {
         this.props.showAboutModalMacOS();
       }
     });
-  }
-
-  beforeWindowUnload(event) {
```

Comment:
Removed the whole block as it seemed to have no use. Also, according to [this thread](https://github.com/electron/electron/issues/2301), it's not recommended to override this event.

No full review is needed for the PR yet.